patches/api/0382-Add-PlayerReleaseItemEvent.patch

```diff
@@ -0,0 +1,64 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: u9g <git@u9g.dev>
+Date: Tue, 3 May 2022 20:41:30 -0400
+Subject: [PATCH] Add PlayerReleaseItemEvent
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/player/PlayerReleaseItemEvent.java b/src/main/java/com/destroystokyo/paper/event/player/PlayerReleaseItemEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..b3b85d54090c567ae05bf68809c47c24ea036e8a
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/player/PlayerReleaseItemEvent.java
+@@ -0,0 +1,52 @@
++package com.destroystokyo.paper.event.player;
++
++import org.bukkit.entity.Player;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.player.PlayerEvent;
++import org.bukkit.inventory.ItemStack;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Called when the server detects a player releasing an item.
```

Comment:
should probably be better clarified to say that this is "stopped using", idk exactly how to word this but I don't think that this really clarifies what the event is actually covering too well here

Reply:
renamed event to `PlayerStopUsingItemEvent`